admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -92,9 +92,9 @@
 
       <div class="{if $display_header}content-div{/if} {if !isset($page_header_toolbar)}-notoolbar{/if} {if $current_tab_level == 3}with-tabs{/if}">
 
-        {hook h='displayAdminAfterHeader'}
-
         {if $display_header}
+          {hook h='displayAdminAfterHeader'}
+
```

Comment:
Isn't a BC ?

Reply:
Ping @PrestaShop/prestashop-core-developers ?

Reply:
Seems logic to me to trigger displayAdminAfterHeader only if header is displayed

Do I just have to notice it in the PR description ?

Reply:
Yes